Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/audit/transform/AuditTransformsTest.java

```diff
@@ -190,7 +190,7 @@ protected void testGetActiveParticipantSource(LogEventRequestType request, Boole
         }
 
         assertEquals("NetworkAccessPointID mismatch", ipOrHost, sourceActiveParticipant.getNetworkAccessPointID());
-        assertEquals("SourceActiveParticipant requestor flag mismatch", isRequesting,
+        assertEquals("SourceActiveParticipant requestor flag mismatch", Boolean.TRUE,
```

Comment:
Discussed w/ @alameluchidambaram offline -- this is a bug fix for a scenario we were not testing before.  "sourceActiveParticipant.isUserIsRequestor()" does not relate to "isRequesting".

As a side note, we may want to consider renaming "isRequesting" to explain WHAT is requesting, but that's outside the scope of this ticket.